tests/test_tutorial/test_metadata/test_tutorial001.py

```diff
@@ -7,9 +7,19 @@
 openapi_schema = {
     "openapi": "3.0.2",
     "info": {
-        "title": "My Super Project",
-        "version": "2.5.0",
-        "description": "This is a very fancy project, with auto docs for the API and everything",
+        "title": "My API",
+        "version": "0.0.1",
+        "description": "This is my API, with auto docs for the API and everything",
```

Comment:
Is there a reason for changing the fields `title`, `version` and `description` here?

Reply:
Yes, the title and description did not reflect what they should be (`My Super Project`) imo.